packages/middleware/src/types/common.ts

```diff
@@ -22,6 +22,14 @@ export type ResponseWithAlokaiLocals = Response<
     [key: string]: any;
   }
 >;
+export type RequestWithFiles = Request & {
```

Comment:
I am afraid if we decide to enhance the Request object even further in the future, we're gonna have a problem with that type's name.

Currently it's just Request + Files but perhaps we will add another request-enhancing middleware in the future (e.g. [session](https://github.com/expressjs/session)). Then we will have to rename it to `RequestWithFilesAndSession`) and the more middlewares we add, the longer the name it will become.

Can we use this intersection type in line 107?

Reply:
Or maybe we can just come up with some more generic name, e.g. `AlokaiRequest`